components/textfield/stories/template.js

```diff
@@ -109,11 +125,11 @@ export const Template = ({
 		}))}
 		${when(multiline,
 			() => html`<textarea
-				placeholder=${ifDefined(placeholder)}
+				placeholder=${ifDefined(placeholder && placeholder.trim() ? placeholder : undefined)}
```

Comment:
This change makes it so that if there is no string defined for `placeholder` that the attribute doesn't show up in the markup

Reply:
Great idea! Want to do that for `pattern` too? There's a couple instances where that could be handy.

```
pattern=${ifDefined(pattern ? pattern : undefined)}
```